jvm-packages/xgboost4j/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,9 @@
+# Set root logger level to DEBUG and its only appender to A1.
+log4j.rootLogger=DEBUG, A1
```

Comment:
would you mind providing more details about the error? I don't think a missing log4j.properties file would cause the program crash?